web/tests/left_sidebar_navigation_area.test.cjs

```diff
@@ -132,14 +126,11 @@ run_test("update_count_in_dom", () => {
 
     assert.equal($("<mentioned-count>").text(), "222");
     assert.equal($("<home-count>").text(), "333");
+    assert.equal($("<condensed-unread-count>").text(), "333");
     assert.equal($("<starred-count>").text(), "444");
-    assert.equal($("<scheduled-count>").text(), "555");
-    assert.equal($("<reminders-count>").text(), "888");
+    assert.equal($("<scheduled-count>").text(), "");
+    assert.equal($("<reminders-count>").text(), "");
```

Comment:
I don't know why these changed, but they feel unrelated to channel folders things, but it happened while the test was disabled